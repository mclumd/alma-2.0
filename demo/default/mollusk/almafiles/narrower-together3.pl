rel(mollusk, manny).
rel(nautilus, nancy).
rel(naked_nautilus, barry).
