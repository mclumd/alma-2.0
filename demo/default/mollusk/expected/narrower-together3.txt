rel(mollusk, manny)
rel(shell_bearer, manny)
rel(nautilus, nancy)
rel(cephalopod, nancy)
rel(mollusk, nancy)
abnormal(nancy, cephalopod, "rel(shell_bearer, nancy)")
rel(shell_bearer, nancy)
rel(naked_nautilus, barry)
rel(nautilus, barry)
rel(cephalopod, barry)
rel(mollusk, barry)
abnormal(barry, mollusk, "~rel(shell_bearer, barry)")
abnormal(barry, nautilus, "~rel(shell_bearer, barry)")
~rel(shell_bearer, barry)
